Makefile

```diff
@@ -201,11 +201,13 @@ godocs:
 
 build-docs:
 	@cd docs && \
-	while read p; do \
-		(git checkout $${p} && npm install && VUEPRESS_BASE="/$${p}/" npm run build) ; \
-		mkdir -p ~/output/$${p} ; \
-		cp -r .vuepress/dist/* ~/output/$${p}/ ; \
-		cp ~/output/$${p}/index.html ~/output ; \
+	while read -a p; do \
+		branch=$${p[0]} ; \
+		path_prefix=$${p[1]} ; \
+		(git checkout $${branch} && npm install && VUEPRESS_BASE="/$${path_prefix}/" npm run build) ; \
```

Comment:
Is there a way to test this before merging? this will `git checkout master`, and then I'm not sure if `versions` still contains the new v0.39 line. Anyways, my `output` folder only contained a `master` subfolder.

Reply:
Testing should work normally. I think the issue for you may have been if you were testing before #7412 was merged, then the building of v0.39 would have failed.

The `while read` should load the `versions` file and be reading line-by-line from whatever branch you ran `make build-docs` from. I'm not the biggest fan of this logic, but didn't want to change it as it seems to work for me, and i think we are doing the same procedure for tendermint docs too.

Can you try git fetch (maybe need to `git checkout launchpad/backports && git pull`?) now that `launchpad/backports` branch is compiling docs properly?